WNPRC_Purchasing/src/org/labkey/wnprc_purchasing/WNPRC_PurchasingController.java

```diff
@@ -296,7 +296,7 @@ private void sendRequestChangeEmailNotification(String oldStatus, EmailTemplateF
                     Map<Integer, String> purchasingDirUserIds = getPurchasingDirectorUserIds();
 
                     //get folder admin users
-                    List<User> adminUsers = SecurityManager.getUsersWithPermissions(getContainer(), Collections.singleton(AdminPermission.class));
```

Comment:
we should keep the 21.7 version

Reply:
@gottfred  Can you look at this change?

Reply:
This was a patch that Josh asked me to apply in this ticket: https://www.labkey.org/WNPRC/support%20tickets/issues-details.view?issueId=44448

So this change is what we want